### DISCUSSION TITLE
It is possible to create a custom field with dynamic dropdown?

### DISCUSSION THREAD

**Discussion by tiicaa** [liedekef/events-made-easy#148] [Q&A]
I made a custom dropdown field, this custom field was inserted in the RSVP form, the user must choose a date in this dropdown. My question is: would it be possible for the user to select and somehow the selected date to disappear so that the next user can only see the remaining dates?

**Comment by liedekef** [comment]
Sorry, but this is not possible without creating a filter, going through all the bookings, checking the answers and then changing the field options. This requires extensive me knowledge, so in short: not possible.
